tee-worker/service/src/main.rs

```diff
@@ -872,3 +828,47 @@ fn check_we_are_primary_validateer(
 		node_api.enclave_count(Some(*register_enclave_xt_header.parent_hash()))?;
 	Ok(enclave_count_of_previous_block == 0)
 }
+
+fn data_provider(config: &Config) -> DataProvidersStatic {
+	let build_in_mode = vec!["dev", "mock", "prod", "staging"];
+	let build_in_config: Value =
+		serde_json::from_slice(include_bytes!("running-mode-config.json")).unwrap();
+
+	let mut data_provider_config = if build_in_mode.contains(&config.running_mode.as_str()) {
```

Comment:
So if I understand it correctly, `running-mode` should be either one of the built-in modes, or a file path?

Reply:
Yes